client/components/autocomplete2/index.jsx

```diff
@@ -66,11 +68,32 @@ export class Autocomplete extends Component {
     })
   }
 
-  onSelect = (item) => {
-    const { onSelect, items } = this.props
-    items.push(item._id)
-    onSelect(uniq(items))
-    this.onBlur()
+  getResItem = async (selected) => {
```

Comment:
What is res in `getResItem` here? Is it result? If so, we should probably just use the term to avoid confusion. Same goes for `resObject`

Reply:
Makes sense -- just updated naming to better reflect what things are used for